bill-of-materials.json

```diff
@@ -188,6 +206,15 @@
 			}
 		]
 	},
+	{
+		"project": "github.com/grpc-ecosystem/grpc-gateway/v2",
```

Comment:
We don't bump grpc-gateway in 3.5. It's a big change. 

https://github.com/etcd-io/etcd/pull/16595

Reply:
It should be fine to just introduce an indirect dependency on grpc-gateway/v2. It means we are still using grpc-gateway v1 in 3.5